assets/js/script.js

```diff
@@ -81,20 +83,27 @@ function searchIMDB(q) {
                 $("#watch-trailer").addClass("hide");
                 $("#add-movie").addClass("hide");
                 $("#cardHolder").show();
+                $(".cardBtnHolder").hide(); // hide add button on error
+                
             }
             else {
                 $("#movie-title").text(data.Title);
                 $("#movie-plot").text(data.Plot);
                 $("#watch-trailer").addClass("visible");
                 $("#add-movie").addClass("visible");
-                searchYouTube(q + " " + data.Year);
+                $("#cardHolder").show();
+                $(".cardBtnHolder").show(); // show add button
+                // searchYouTube(q + " " + data.Year); // comment it out when youtube quote is full
```

Comment:
I wonder if we should keep it uncommented out when committing to GitHub, and then make the change if it ends up affecting you?

Reply:
oops sorry I forgot to put it back. Had to disable youtube so I could do testing.

Reply:
Sure thing I made the change during the merge!

Reply:
yay!! Thank you!